packages/app-project/src/screens/SubjectTalkPage/components/SubjectTalkData/components/Tags/TagsContainer.jsx

```diff
@@ -0,0 +1,278 @@
+import { string } from 'prop-types'
+import { useState } from 'react'
+
+import {
+  addTagVote,
+  createVotableTag,
+  removeTagVote
+} from '@helpers'
+
+import {
+  useTags,
+  useTagVotes,
+  useVotableTags
+} from '@hooks'
+
+import Tags from './Tags'
+
+function TagsContainer({
+  projectId,
+  subjectId,
+  userId
+}) {
+  const [loading, setLoading] = useState(false)
```

Comment:
We chatted about this edge case in-person where we want to prevent someone from double-clicking a votable tag button too quickly, so the existence of this state variable make sense. However, I think the name of it could easily be confused with a loading state for _all_ of the hooks below it. Can you rename this to something like `tagsRequestInProgress`?